### ISSUE TITLE
Voting on Features

### ISSUE THREAD

**Issue by ejmr** [emacs-php/php-suite#5]
This is related to issue #2 for creating a wishlist.  Does anyone here have experience using [FeatHub](https://feathub.com/)?  If so, I would greatly appreciate any opinions on it.  For people like myself who have never used it, how does @emacs-php feel about potentially using FeatHub as a way to help prioritize features for PHP Mode and all related projects?

**Comment by zonuexe** [comment]
@ejmr 
Thanks for the suggestion. I have imported php-suite into FeatHub.
https://feathub.com/emacs-php/php-suite

I may not be able to realize all the requests, but I would like to know the demand of users.

[![Feature Requests](http://feathub.com/emacs-php/php-suite?format=svg)](http://feathub.com/emacs-php/php-suite)

**Comment by zonuexe** [comment]
I do not seem to be able to see badges from my environment. If it becomes visible after tomorrow, it applies to other projects as well.

**Comment by ejmr** [comment]
@zonuexe 

> I may not be able to realize all the requests, but I would like to know the demand of the users.

I knew you would be a great maintainer, and that statement proves it.  I think that I am going to try to contribute again---I've decide I woud like to publish more code before I die sometime in the next eighteen months.  And one of the major reasons I am compelled to contribute once again is because of your talent as a manatiner.

PHP Mode is in great hands, and I couldn't be happier with your work.

`--ejmr`

**Comment by cyrialize** [comment]
@zonuexe You're the best man, I love how much you've contributed to the state of PHP on Emacs. Emacs has made me get into open source, but it's stuff like PHP-suite that encourages me to master elisp.

**Comment by ejmr** [comment]
@jonnycyr 

Yeah, creating @emacs-php and moving multiple PHP modules for GNU Emacs into one place might be the best thing @zonuexe has done yet.  It is certainly a fantastic idea which I never thought of once during my six years as the maintainer.  I love how quickly Usami as the maintainer organized everything.

`--ejmr`
